app/graphql/types/scalar/any_primative_type.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```

Comment:
can we rename this AnyPrimitiveType after the merge?

Reply:
I will do that right now!

Reply:
Thanks!